packages/cli/src/util/Cutter.ts

```diff
@@ -35,7 +35,7 @@ export default class Cutter {
 
     const cmd = path.join(target, 'init.sh');
     if (fs.existsSync(cmd)) {
-      const { stderr } = await exec(`bash ${cmd}`);
+      const { stderr } = await util.promisify(_execFile)('bash', [cmd]);
```

Comment:
this is a nice change but I believe this conditional is dead code! init.sh was added in the original CLI implementation but doesn't seem to exist any longer. suggest deleting the whole conditional.  /cc @ben-harris-5